dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.json

```diff
@@ -0,0 +1,476 @@
+    },
+    {
+     "type": "rating",
+     "name": "recruitResponsibility",
+     "title": "Recruit Responsibility. 1. recruit representative population    --- 5. recruit students interested in CS",
+     "titleLocation": "hidden",
+     "minRateDescription": "Strongly aligned with A",
+     "maxRateDescription": "Strongly aligned with B"
+    },
+    {
+     "type": "html",
+     "name": "clickTo119",
+     "html": "<p><span style=\"font-size: 14pt; font-family: verdana, geneva, sans-serif;\"><strong>Overall Workshop Feedback</strong></span></p>\n<p>We’d like to hear your feedback about your experience this week overall, <Strong>especially how this professional development could better fit your learning needs, or how it could be improved</strong>.\n"
+    },
+    {
+     "type": "comment",
+     "name": "activities_you_liked",
+     "title": "What were the one or two things you liked most about the activities you did in this workshop and why?",
+     "cols": 6,
+     "rows": 6
+    },
+    {
+     "type": "comment",
+     "name": "activities_youd_change",
+     "title": "What are the one or two things you would change about the activities you did in this workshop?\t",
+     "cols": 60,
+     "rows": 6
+    },
+    {
+     "type": "paneldynamic",
+     "name": "facilitator_feedback_panel",
+     "title": "Facilitator Feedback",
+     "description": "Please leave feedback about your facilitator(s).",
+     "templateElements": [
+      {
+       "type": "matrix",
+       "name": "facilitator_effectiveness",
+       "title": "During my workshop, {panel.facilitatorName} did the following:",
```

Comment:
you can keep the facilitator variables as they are here (ex don't change to {panel.facilitator_name})